### PR TITLE
Remove voltage level and the branches it is connected to

### PR DESCRIPTION
**Pull request by etiennehomer**
Use RemoveVoltageLevel modification of powsybl-core for voltage level removal.

Signed-off-by: Etienne Homer <etiennehomer@gmail.com>

### CODE REVIEW

**Comment by etiennehomer** [comment]
TODO: remove powsybl version override

File: src/test/java/org/gridsuite/modification/server/modifications/VoltageLevelDeletionTest.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+                .equipmentType("VOLTAGE_LEVEL")
+                .equipmentId("v1")
+                .build();
+    }
+
+    @Override
+    protected ModificationInfos buildModificationUpdate() {
+        return EquipmentDeletionInfos.builder()
+                .equipmentType("VOLTAGE_LEVEL")
+                .equipmentId("v1")
```
Comment (FranckLecuyer) [inline src/test/java/org/gridsuite/modification/server/modifications/VoltageLevelDeletionTest.java:47]: Set values in equipmentType and equipmentId, different from the values set in buildModification method just above

File: src/test/java/org/gridsuite/modification/server/modifications/VoltageLevelDeletionTest.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    }
+
+    @Override
+    protected void assertNetworkAfterDeletion() {
+        assertNotNull(getNetwork().getVoltageLevel("v1"));
+    }
+
+    @SneakyThrows
+    @Test
+    public void testRemoveVoltageLevel() {
```
Comment (FranckLecuyer) [inline src/test/java/org/gridsuite/modification/server/modifications/VoltageLevelDeletionTest.java:68]: You can just move all the assertions in this test to assertNetworkAfterCreation method, and then remove this test method ...

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]
Code review : see remarks
Tests: 
Pb when displaying a nearby voltage level of voltage level deleted, when there was a line in bypass ('pontage') in the nearby voltage level leading to the voltage level deleted : the following error message is displayed : "The diagram couldn't be loaded"

File: src/test/java/org/gridsuite/modification/server/modifications/VoltageLevelDeletionTest.java (hunk 1 above)
Comment (etiennehomer) [reply src/test/java/org/gridsuite/modification/server/modifications/VoltageLevelDeletionTest.java]: Done

File: src/test/java/org/gridsuite/modification/server/modifications/VoltageLevelDeletionTest.java (hunk 2 above)
Comment (etiennehomer) [reply src/test/java/org/gridsuite/modification/server/modifications/VoltageLevelDeletionTest.java]: done

**Review by EstherDarkish — APPROVED** [review]
Tests OK, code OK

**Pull request merged**